src/coreclr/jit/lower.h

```diff
@@ -447,6 +444,30 @@ class Lowering final : public Phase
         return m_lsra->isContainableMemoryOp(node);
     }
 
+    // Return true if 'childNode' is a containable memory op by its size relative to the 'parentNode'.
+    // Currently very conservative.
+    bool IsContainableMemoryOpSize(GenTree* parentNode, GenTree* childNode) const
```

Comment:
Not sure what to name this. `IsContainableMemoryOpSize` might be confusing considering we have a `IsContainableMemoryOp` already.